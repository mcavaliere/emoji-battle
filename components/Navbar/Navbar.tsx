import React from 'react';
import { Heading, Center, VStack } from '@chakra-ui/react';

export function Navbar(): JSX.Element {
  return (
    <Center
      bg="green.100"
      padding={{ base: 3, lg: 5 }}
      display={{ base: 'flex' }}
    >
      <VStack>
        <Heading size="md">Emoji 🤪 ⚔️ 😀 Battle</Heading>
        <Heading size="sm">🥋Welcome to the dojo.</Heading>
      </VStack>
    </Center>
  );
}
