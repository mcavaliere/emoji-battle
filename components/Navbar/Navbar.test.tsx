import React from 'react';
import { render } from '@testing-library/react';
import { Navbar } from './Navbar';

describe('Navbar', () => {
  it('renders correctly', () => {
    const { container } = render(<Navbar myProp="foo" />);
    expect(container).toMatchSnapshot();
  });
});