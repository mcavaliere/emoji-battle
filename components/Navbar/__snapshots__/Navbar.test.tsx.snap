// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div>
  <div
    class="css-15ojgsl"
  >
    <h2
      class="chakra-heading css-0"
    >
      Navbar Created.
    </h2>
  </div>
</div>
`;
